proto/storage/alert.proto

```diff
@@ -14,21 +14,21 @@ option java_package = "io.stackrox.proto.storage";
 
 message Alert {
   message Deployment {
-    string id = 1 [(gogoproto.moretags) = 'search:"Deployment ID,store,hidden" sql:"index=hash,type(uuid)"'];
-    string name = 2 [(gogoproto.moretags) = 'search:"Deployment,store"'];
+    string id = 1 []; // @gotags: search:"Deployment ID,store,hidden" sql:"index=hash,type(uuid)"
+    string name = 2 []; // @gotags: search:"Deployment,store"
     string type = 4;
     string namespace = 5; // This field has to be duplicated in Alert for scope management and search.
     string namespace_id = 16; // This field has to be duplicated in Alert for scope management and search.
     map<string, string> labels = 7 [(gogoproto.moretags) = 'sensorhash:"ignore"'];
     string cluster_id = 9; // This field has to be duplicated in Alert for scope management and search.
     string cluster_name = 10; // This field has to be duplicated in Alert for scope management and search.
     message Container {
-      ContainerImage image = 3 [(gogoproto.moretags) = 'search:"-" sql:"ignore-fks,ignore-index"'];
+      ContainerImage image = 3 []; // @gotags: search:"-" sql:"ignore-fks,ignore-index"
```

Comment:
Why do we have the empty `[]`?

(I'm no proto expert so it could be valid, but it certainly looks odd)

Reply:
Fixed https://github.com/janisz/buf/commit/9fd7c64fa75582d90c49b39d5a51e4017b10edeb#diff-1cd9512c882eae652bd4295025d5939c1809056cfcc005c387943df26a863231R859